code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/subs.cljs

```diff
@@ -34,7 +34,7 @@
   ::deployment-set
   :<- [::deployment-set-stored-and-edited]
   (fn [[stored edited]]
-    (merge stored edited)))
```

Comment:
Is this change related to this PR?

Reply:
Yes. After executing an action, e.g Start,the Cancel button was not enabled in the ui, even though the operation was returned from the server. I found that it was because the `operations` attribute in `edited` was overriding the `operations` attribute in stored. So I changed the merge operation to only include from `edited` the attributes that can be edited by the user, and that fixed this issue .